sdk/timeseriesinsights/remove-test-resources-pre.ps1

```diff
@@ -0,0 +1,53 @@
+# IMPORTANT: Use the contents of this file as a template on what resources should be cleaned in the test subscription
```

Comment:
This file is no longer needed. This was created to get around the ADT delete issue when there was an eventhub endpoint configured. ADT has since resolved that issue and we no longer need this. I think @sindhu-nagesh was in the process of removing it from ADT as well.